### DISCUSSION TITLE
can i play video without the widget tree?

### DISCUSSION THREAD

**Discussion by ymmmvp36** [alexmercerind/dart_vlc#141] [Q&A] [answered]
hi,can i play video without the widget tree?i just want to play in a new native window like the early version?

**Comment by alexmercerind** [comment]
This isn't actually configurable presently.

**Comment by alexmercerind** [comment, marked as answer]
Create `Player` instance by passing `VideoDimensions` as `0`.

```dart
Player player = Player(
  id: 1234,
  videoDimensions: const VideoDimensions(0, 0),
);
```
